file-formats/ilmb/type/zif_aff_ilmb_v1.intf.abap

```diff
@@ -0,0 +1,83 @@
+INTERFACE zif_aff_ilmb_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Condition Field</p>
+    "! Condition field
+    BEGIN OF ty_cond_field,
+      "! <p class="shorttext">Name</p>
+      "! Condition field
+      name             TYPE zif_aff_types_v1=>ty_object_name_30,
```

Comment:
most probably the name of the condition field is required. Most probably further fields need to be required.

```suggestion
      "! $required
      name             TYPE zif_aff_types_v1=>ty_object_name_30,
```

Reply:
That's right. Have made them mandatory.